manager-bundle/src/Api/Command/VersionCommand.php

```diff
@@ -32,11 +32,14 @@ public function __construct(private readonly Application $application)
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $output->write(json_encode([
-            'version' => Application::VERSION,
-            'commands' => $this->getCommandNames(),
-            'features' => $this->getFeatures(),
-        ]));
+        $output->write(json_encode(
+            [
+                'version' => Application::VERSION,
+                'commands' => $this->getCommandNames(),
+                'features' => $this->getFeatures(),
+            ],
+            JSON_THROW_ON_ERROR)
+        );
```

Comment:
```suggestion
            JSON_THROW_ON_ERROR,
        ));
```

Reply:
There is no trailing comma for function arguments. We only have this rule for constructor property promotion.